nginx.conf

```diff
@@ -0,0 +1,35 @@
+worker_processes auto;
+pid /run/nginx.pid;
+error_log /var/log/nginx/error.log;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+  worker_connections 768;
+}
+
+http {
+  sendfile on;
+  tcp_nopush on;
+  types_hash_max_size 2048;
+
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3; # Dropping SSLv3, ref: POODLE
+  ssl_prefer_server_ciphers on;
+
+  access_log /var/log/nginx/access.log;
+  gzip on;
+  include /etc/nginx/conf.d/*.conf;
+  server {
+    listen 8080;
+
+    index index.html index.htm index.nginx-debian.html;
+    server_name _;
+    location / {
+      root /var/www/html/browser;
+      try_files $uri $uri/ /index.html;
```

Comment:
this is not safe, you allow render any file passed as $uri, so if you have something like .env with secret values inside - it will be rendered as open text.

learn how to use this
https://nginx.org/en/docs/http/ngx_http_core_module.html#try_files

google has these tutorials and everyone is copying without analyzing what exactly they had done. And the impact, that will cause
- security breaches
- downtime

When you have a configuration, it's not enough that it **worked** you need to know how/why it has worked. Otherwise you will become a chatgpt victim

find a better way